src/buf_encoder.rs

```diff
@@ -58,7 +58,7 @@ impl<const CAP: usize> BufEncoder<CAP> {
 
     #[inline]
     #[track_caller]
-    fn put_bytes_inner<I>(&mut self, bytes: I, case: Case)
+    pub(crate) fn put_bytes_inner<I>(&mut self, bytes: I, case: Case)
```

Comment:
I don't think this is needed. Just calling `put_bytes` with an iterator works already.